### DISCUSSION TITLE
Sound cards, devices, capabilities

### DISCUSSION THREAD

**Discussion by Torinde** [86Box/86Box#4759] [General]
Sharing here the Mobygames list of sound [devices](https://www.mobygames.com/attributes/tech-specs/1/) / [capabilities](https://www.mobygames.com/attributes/tech-specs/27/) and the number of DOS/Win9x games utilizing those.
You can tick /untick the boxes above for cards that are (not) already emulated by 86Box. I filled in what I found. Related: https://github.com/dosbox-staging/dosbox-staging/discussions/3912

- Sound Blaster - 5132
  - [x] Sound Blaster - 2392
  - [x] Sound Blaster Pro - 1095+2
  - [x] Sound Blaster 16 - 934+131
    - [x] [CQM](https://github.com/joncampbell123/dosbox-x/issues/4621)
    - [ ] [ASP/CSP](https://github.com/joncampbell123/dosbox-x/issues/62)
  - [x] Sound Blaster AWE32 - 543+35
- Adlib - 1955
  - [x] Adlib - 1647
  - [x] Adlib Gold - 308
- [x] [Ensoniq Soundscape](https://www.mobygames.com/attributes/attribute/77/) - 268 - AudioPCI is emulated and supports Ensoniq Soundscape? #933 
- [x] DirectX Compatible Sound Card - 209
- [x] Microsoft Sound System - 189+4
- [x] Game Blaster (CMS) - 100
- [x] ESS AudioDrive - 85
- [x] Sound Galaxy NX Pro 16 - 33
- [x] Innovation Sound Standard - 15
- Media Vision - 894
  - [x] [Thunderboard](https://www.mobygames.com/attributes/attribute/81/) - 126
  - [x] [Pro Audio Spectrum](https://www.mobygames.com/attributes/attribute/79/) - 625
  - [x] Pro Audio Spectrum Plus - 40 - Pro Audio Spectrum Plus = Pro Audio Spectrum + Thunderboard?
  - [x] Pro Audio Spectrum 16 - 103
- Surround sound - 620 for Win9x and more for WinXP+
  - [ ] #4648 
  - [ ] [Dolby Digital (AC-3) audio coding 5.1](https://www.mobygames.com/attributes/attribute/274/) - 374 
  - [ ] [Dolby Pro Logic II](https://www.mobygames.com/attributes/attribute/206/) - 36
  - [ ] [Dolby Surround](https://www.mobygames.com/attributes/attribute/136/) - 210
  - [ ] Dolby/DTS other encodings for 5.1, 6.1, 7.1 for WinXP and up
  - [ ] [THX](https://www.mobygames.com/attributes/attribute/1611/) - 10 games - [THX TruStudio](https://sg.creative.com/corporate/pressroom?id=13137)?
- [3D sound](https://github.com/kcat/dsoal/issues/36#issuecomment-2323365888) - 1308
  - [ ] [Aureal3D](https://www.mobygames.com/attributes/attribute/1273/) - 69 - #2911 
  - [x] [DirectSound3D](https://www.mobygames.com/attributes/attribute/1274/) - 471 - AudioPCI? alternative [DSOAL](https://github.com/kcat/dsoal) and [OpenAL-Soft](https://github.com/kcat/openal-soft)
  - [x] [EAX](https://www.mobygames.com/attributes/attribute/1271/) - 333 - AudioPCI? alternative [DSOAL](https://github.com/kcat/dsoal) and [OpenAL-Soft](https://github.com/kcat/openal-soft)
  - [ ] [EAX Advanced HD](https://www.mobygames.com/attributes/attribute/1272/) - 374 - alternative [DSOAL](https://github.com/kcat/dsoal) and [OpenAL-Soft](https://github.com/kcat/openal-soft)
  - [ ] [I3DL2](https://www.iasig.org/index.php/projects/projects-menu/21-3d-audio-wg) ([2 exclusive](https://docs.google.com/document/d/1SyOw7D0H2ecGxzPsCFEFKwupU8LuazUrlkI-gVTpyhc/edit#heading=h.nu0xc1c2nmo4)) - [not yet supported](https://github.com/kcat/dsoal/issues/36#issuecomment-2318423370) by DSOAL
  - [x] [OpenAL](https://www.mobygames.com/attributes/attribute/1276/) - 150 - AudioPCI? alternative [DSOAL](https://github.com/kcat/dsoal) and [OpenAL-Soft](https://github.com/kcat/openal-soft)
  - [ ] IBM [MWave](https://en.wikipedia.org/wiki/Mwave) - ISA 1992, PCI 1998, MCA, [3D](https://www.vogons.org/viewtopic.php?t=66659) - used in IBM Aptiva (#2248) and others
  - [x] Gravis Ultrasound MAX - ISA 1994 "[software CD](https://en.wikipedia.org/wiki/Gravis_UltraSound#UltraSound_MAX) includes a demo that featured "3D holographic sound" through the use of software HRTF filters." 86Box developer only?
  - QSound - 47 games [1](https://www.qsound.com/partners/gaming/pc-games.htm), [2](https://www.uvlist.net/groups/games-list/qsound), [QMixer](https://docs.google.com/document/d/1SyOw7D0H2ecGxzPsCFEFKwupU8LuazUrlkI-gVTpyhc/edit#heading=h.nu0xc1c2nmo4) - 2
    - [ ] ISA 1992 SB16 [ASP/CSP](https://github.com/joncampbell123/dosbox-x/issues/62)
    - [ ] ISA 1994 [AWE32 3D](https://www.ibiblio.org/GameBytes/issue19/greviews/sbawe32.html) 
    - [ ] PCI 1997 [Trident Q3D 2.0](https://en.wikipedia.org/wiki/Trident_Microsystems#Sound_chipsets) 
    - [ ] PCI 2000 [ESS1948](https://dosdays.co.uk/topics/Manufacturers/ess.php)
  - SRS: 
    - [ ] ISA 1994 Media Vision [Pro3D/Premium3D](https://dosdays.co.uk/topics/Manufacturers/mediavision.php#Pro3D) - supports [VBE/AI](https://en.wikipedia.org/wiki/VESA_BIOS_Extensions#Audio_interface_extensions_(AI))
    - [ ] ISA 1997 Crystal [CS4237B](https://dosdays.co.uk/media/crystal/CS4237B_datasheet.pdf) (chip emulated by [86Box](https://emulation.gametechwiki.com/index.php/86Box#Audio)) - [AOpen 97353-2.1, Wavetable connector](https://www.ebay.com/itm/255765875898)
  - Sensaura - [GameCODA](https://docs.google.com/document/d/1SyOw7D0H2ecGxzPsCFEFKwupU8LuazUrlkI-gVTpyhc/edit#heading=h.nu0xc1c2nmo4) - 2 exclusive
    - [ ] PCI 1999 Crystal [CS4630](https://www.alldatasheet.com/datasheet-pdf/view/58126/CIRRUS/CS4630.html) - Turtle Beach Santa Cruz, Hercules Game Theater XP
    - [ ] PCI 1999 ESS Canyon3D - [MX400](https://dosdays.co.uk/topics/Manufacturers/diamond.php#MX400), [I/O Magic Typhoon](https://web.archive.org/web/20020213031442/https://iomagic.com/support/soundcards/typhoon_theater/index.htm) supporting also [I3DL2](https://www.iasig.org/index.php/projects/projects-menu/21-3d-audio-wg) ([2 exclusive](https://docs.google.com/document/d/1SyOw7D0H2ecGxzPsCFEFKwupU8LuazUrlkI-gVTpyhc/edit#heading=h.nu0xc1c2nmo4))
    - [ ] PCI 2001 [ES1992](https://pdf.datasheetcatalog.com/datasheet_pdf/ess-technology/ES1992S.pdf) Canyon3D-2
    - [ ] PCI/AC'97 2001 [Soundstorm](https://www.nvidia.com/en-us/drivers/nforce-udp-win9x-427/)
    - [ ] [PCI/AC'97 chips](https://en.wikipedia.org/wiki/Sensaura#Compatible_hardware[14]) - Yamaha, Analog Devices, C-Media, Realtek, VIA
  - [ ] [EarSound IAS](https://dosdays.co.uk/topics/3daudio_games.php) - 11 games, including some exclusively supporting that standard such as Quake II, Civ2, Mechwarrior2
- MIDI - 1710
  - [x] General MIDI - 371
  - [x] Generic MPU-401 MIDI devices - 120
  - [x] Roland MT-32 (and LAPC-I) - 915
  - [ ] [Roland RAP-10](https://www.mobygames.com/attributes/attribute/87/) - 185 - [ISA card](https://en.wikipedia.org/wiki/Roland_Sound_Canvas#Sound_Canvas_Personal_Computer_Products), includes two-channel 16-bit digital audio support.
  - [x] [Roland Sound Canvas](https://www.mobygames.com/attributes/attribute/572/) - 119 - [X](https://github.com/joncampbell123/dosbox-x/issues/4995), [Staging](https://github.com/dosbox-staging/dosbox-staging/issues/2138) preparing a [NukedSC55 CLAP plugin](https://github.com/nukeykt/Nuked-SC55/issues/91#issuecomment-2311548283)
  - [ ] Roland SC-88 / JV-1080 - [15](https://www.vogons.org/viewtopic.php?p=1190364#p1190364)
  - [ ] [Casio MT-540/CT-460/CSM-1](https://nerdlypleasures.blogspot.com/2012/04/unique-pc-hardware-game-support.html) - 14 - More [info and samples](https://www.vogons.org/viewtopic.php?t=49707) of how it sounds, [Space Quest 3 sounds comparison](https://www.youtube.com/watch?v=ICX6hK3Se6k). Reddit [emulation question](https://www.reddit.com/r/dosgaming/comments/ttj33p/casio_mt540_emulation_for_space_quest_3/?rdt=64299).
  - [ ] Yamaha XG/MU - [X](https://github.com/joncampbell123/dosbox-x/issues/3632)
  - [ ] [General MIDI Level 2](https://en.wikipedia.org/wiki/General_MIDI_Level_2)
- Covox Master - 238
  - [x] [Covox Sound Master](https://www.mobygames.com/attributes/attribute/80/) - 157 - [X](https://github.com/joncampbell123/dosbox-x/issues/2396), [Staging](https://github.com/dosbox-staging/dosbox-staging/issues/2219)
  - [x] [Covox Sound Master Plus](https://www.mobygames.com/attributes/attribute/1725/) - 17 - "Adlib clone with the ability output digitized sound via direct drive methods." [1](https://nerdlypleasures.blogspot.com/2014/09/the-mysterious-covox-pc-sound-devices.html)
  - [ ] [Covox Sound Master II](https://www.mobygames.com/attributes/attribute/1726/) - 56 - "closer to a Sound Blaster.  It had the OPL2 YM-3812 FM Synthesizer found on the Adlib Music Synthesizer, MIDI in and out via a DE-9 port and a dongle and direct drive and DAC fed digitized sound input and output.  Its MIDI is probably based off the MC6850 ACIA, so it would not be MPU-401 compatible." [1](https://nerdlypleasures.blogspot.com/2014/09/the-mysterious-covox-pc-sound-devices.html)
  - [ ] [VoiceMaster](https://www.mobygames.com/attributes/attribute/768/) - 8 - maybe that's [Covox Voice Master](https://nerdlypleasures.blogspot.com/2014/09/the-mysterious-covox-pc-sound-devices.html)?
- PC Speaker - 2091
  - [x] [regular](https://www.mobygames.com/attributes/attribute/25/) - 1887
  - [ ] [tweaked](https://www.mobygames.com/attributes/attribute/34/) - 204
- [ ] Microphone ([required](https://www.mobygames.com/attributes/attribute/2816/), [supported](https://www.mobygames.com/attributes/attribute/719/) 186) - #3872
- Tandy - 489
  - [x] [Tandy / PCjr](https://www.mobygames.com/attributes/attribute/32/) - 392
  - [x] [Tandy DAC (TL/SL)](https://www.mobygames.com/attributes/attribute/33/) - 97
- Gravis Ultrasound - 634
  - [x] Classic - 542
  - [x] ACE - Classic as add-on card - #3339
  - [x] [MAX](https://www.mobygames.com/attributes/attribute/1045/) - 92 - 86Box developer only?
- [ ] [WaveJammer](https://www.mobygames.com/attributes/attribute/88/) (PCMCIA) - 81 - "Sound Blaster clone PCMCIA card for laptops"
- [x] [PS/1 Audio Card](https://www.mobygames.com/attributes/attribute/86/) - 61 - PS/1 itself is emulated?
- [ ] [ARIA Chipset](https://www.mobygames.com/attributes/attribute/567/) (Prometheus ARIA16?, [1](http://www.faqs.org/faqs/PCsoundcards/aria/faq/), [2](https://www.vogons.org/viewtopic.php?t=65258)) - 61
- [ ] [Reveal FX/32](https://www.mobygames.com/attributes/attribute/573/) - 23 - "can play MT-32/GM/GS music without a TSR and in real-mode DOS." [1](https://www.vogons.org/viewtopic.php?t=45313), [Reveal SoundFX WAVE 32 SC600](https://wiki.preterhuman.net/Reveal_SoundFX_WAVE_32_SC600) has on the box "Soundscape by Ensoniq"
- [ ] [I/O Magic Tempo](https://www.mobygames.com/attributes/attribute/569/) (PCMCIA)- 22 - "OPL3 chip, compatible with Yamaha Gold Sound Standard (GSS). PCMCIA Soundcard with OPL3 and Wav capabilitys?" [1](https://www.vogons.org/viewtopic.php?t=30840&start=460), [2](https://www.vogons.org/viewtopic.php?t=66447&p=749496)
- [ ] [Toptek Golden 16](https://www.mobygames.com/attributes/attribute/570/) - 22 - [Toptek Golden Sound Pro 16 ES](https://wiki.preterhuman.net/Toptek_Multimedia) with TMC chipset and Crystal CS4231?
- [ ] [Wave Blaster (CT1900)](http://www.vgmpf.com/Wiki/index.php/Category:Games_That_Use_Wave_Blaster_For_Music) - 20 - [X](https://github.com/joncampbell123/dosbox-x/issues/4130), #282
- [ ] Wave Blaster II (CT1910)
- [x] IBM Music Feature Card - 19 - emulated by [X](https://github.com/joncampbell123/dosbox-x/pull/4236), [Staging](https://github.com/dosbox-staging/dosbox-staging/pull/2367)
- Parallel port device - 236
  - [x] Disney Sound Source - 138
  - [x] [Covox Speech Thing](https://www.mobygames.com/attributes/attribute/43/) - 56
  - [x] [Speech Thing](https://www.mobygames.com/attributes/attribute/767/) - 8
  - [x] [Intersound MDO](https://www.mobygames.com/attributes/attribute/1860/) - 11 - device distributed by Coktel Vision themselves which is also mentioned on the game manual. The device is almost certainly a custom version of the Covox Speech Thing. [1](https://www.mobygames.com/game/9399/fascination/), testing in 86box with various games shows covox speech thing plays games for it just fine.
  - [ ] [DA Converter](https://www.mobygames.com/attributes/attribute/766/) - 8 - is it the "[ISA DAC](https://86box.net/2025/08/24/86box-v5-0.html)" from 86Box 5.0?
  - [x] [Generic LPT DAC](https://www.mobygames.com/attributes/attribute/45/) - 6
  - [ ] [MSound](https://www.mobygames.com/attributes/attribute/46/) - 9 - more than a simple DAC, however; a DSP contained within the box could be programmed to emulate mono, stereo, or quad stereo output, up to 48KHz. - [Staging](https://github.com/dosbox-staging/dosbox-staging/issues/3902)
- IBM Speech attachment/adapter - 5 - [Staging](https://github.com/dosbox-staging/dosbox-staging/issues/2774)
  - [ ] [PCjr Speech Attachment](https://www.mobygames.com/attributes/attribute/1055/) - 1
  - [ ] [PS/2 Speech Adapter](https://www.mobygames.com/attributes/attribute/134/) - 4
- [x] [SAAYM 8-bit ISA dual Phillips SAA1099 Yamaha YM2151](https://github.com/joncampbell123/dosbox-x/issues/1365) - CMS / GameBlaster Clone with [Nuked-OPM](https://github.com/nukeykt/Nuked-OPM)

**Comment by spc999** [comment]
How do I use yrw801.rom from Yamaha on the rom set?

Is there any way to use it in the DOS support program?

It works on a real card.

Of the OKSORI cards, the PCM series using YMF-278 and yrw801 is actually a card that works on DOS as well.

Oksori 3.8 e PCM 
Oksori WS16 PCM e
Oksori WS16 E PCM
Oksori WS16 PCM III 

![386840741-ef7139e7-088e-4b4c-bc9a-77cea41c5496](https://github.com/user-attachments/assets/bf19eb3c-78c9-4f58-b9cf-6a18679f8894)
![386840745-e4e81a27-29de-4cc2-b379-ee4891c9c615](https://github.com/user-attachments/assets/51ad3849-ae93-417e-a306-93622042dd19)
![386840696-45ad3a18-766f-4b24-9d42-548dc9a23db0](https://github.com/user-attachments/assets/56a92ac5-b1a6-4a9c-8572-8b0b3ccceea9)
![386840726-f2e38210-48f4-47e1-998c-4e9656c6645a](https://github.com/user-attachments/assets/c69bd068-7855-4215-a7a6-9a3e53915089)



[OPL4.zip](https://github.com/user-attachments/files/17793077/OPL4.zip)
[norebang.zip](https://github.com/user-attachments/files/17793078/norebang.zip)


![20200131_203022](https://github.com/user-attachments/assets/c89d3d8e-27ac-4544-bd5b-cae432becad3)
![20200206_214755](https://github.com/user-attachments/assets/c218b123-e276-4f1a-b80a-0653f88d7f78)
![20200311_205325](https://github.com/user-attachments/assets/eb7885ba-5781-45b2-9b10-b4056bf43163)


3.8 is an 8-bit sound card that is not compatible with WSS

WS16 series is WSS compatible

All of the above cards are SoundBlaster Pro compatible and MPU401 compatible.


![1118_203703_SVideo](https://github.com/user-attachments/assets/f4cfc4e4-ac5f-4b92-8d2a-e4dfff546abe)


https://github.com/user-attachments/assets/058913b1-9176-46d8-9e89-bd589266aab9


https://github.com/user-attachments/assets/89e1d0bb-6713-478d-aa1c-a5e37468bc34


https://github.com/user-attachments/assets/430800ea-1c93-4cc0-aafe-09773459ae35



https://github.com/user-attachments/assets/51de8d0b-67f1-4a30-bdd4-4f09a067607c

**Comment by sasamiya-bios** [comment]
Feature Request - 
Sound Blaster 16 / 16PNP / 32 / 32PNP / 64 ISA: Please support disable the on-card IDE controller. The Sound Blaster IDE controller will occupy one IRQ line. Plain Windows 95 does not support PCI IRQ sharing.
And please support connect the IDE CD-ROM drive to the Sound Blaster IDE controller.

**Reply by jriwanek** [reply]
I was already working on variants of the PNP's without IDE.

(Edit) Wanted: ISAPNP Dumps from SB32 PNP and AWE32 PNP cards which *DONT* have an IDE Controller.

**Reply by jriwanek** [reply]
SB16 and AWE64 now have IDEless varients implemented as of build #6386, and some IDE related bug fixes as of #6387, and the EMU 8000 Upgrade card (Goldfinch) as of #6410. No native IDEless AWE32's, but those cover the options between them.

**Reply by Torinde** [reply]
Another Emu-8000 card is [Waveblaster II](https://en.wikipedia.org/wiki/Creative_Wave_Blaster#WaveBlaster_II) (CT1910).

**Comment by sasamiya-bios** [comment]
Add USB Audio -
Windows 98, Windows 98 SE, Windows 2000, Windows Me have built-in driver for USB Audio Class.

**Comment by hjnijlunsing** [comment]
I would love emulation for:
- Gravis Ultrasound PNP featuring the AMD Interwave chip; including 1MB ROM and Maximum of 16MB Ram
- ESFM (perhaps based on: https://github.com/Kagamiin/ESFMu)
- General port passthrough (I.e. I still have a PCI Card ESS-Solo-1 with ESFM chip; would be nice to use this)
- General PCI Passthrough for entire ESS-Solo-1 passthrough as external device (but also GPU's etc.)

**Reply by Torinde** [reply]
Passthrough related:
- #4801

**Reply by hjnijlunsing** [reply]
Another interesting forum: https://forum.mattkc.com/viewtopic.php?t=299

**Reply by win2kgamer** [reply]
ESFM support has been implemented since v4.2 last year and in fact Kagamiin was involved in adding the ESS AudioDrive support to 86Box.

**Comment by sebastien64** [comment]
hello

please add this graphic card (trident 9420)

https://www.vgamuseum.info/index.php/technologies/item/446-trident-tgui9420dgi

thank

**Reply by TC1995** [reply]
datasheets and documents please...

**Reply by TC1995** [reply]
including register/bitblt info.

**Comment by AphisitC** [comment]
Add a New Soundcard - a Yamaha YMF7x4 Family of Soundchip

Yamaha YMF724 (DS-1) - a PCI version of OPL3-SAx chip with Hardware Extended General Midi (Yamaha XG) Wavetable, 2-channel sound, 64-voice polyphony (Hardware), 73 audio streams and Sensaura engine (EAX 2.0 and A3D 1.0 compliant)
[Datasheet for YMF724](https://www.dosdays.co.uk/topics/images/YMF724-ETC.pdf)

YMF740-C (DS-1L) - a cut-down version of YMF724: 32-voice polyphony (Hardware), 41 audio streams, no S/PDIF Out. This chip is used in MoBo (Motherboard) and laptops.
[Datasheet for YMF740-C](https://www.dosdays.co.uk/media/yamaha/YMF740C.pdf)

Yamaha YMF744B (DS-1S) - 4-channel sound + S/PDIF In + all features of YMF724
[Datasheet for YMF744](https://www.dosdays.co.uk/media/yamaha/YMF744.pdf)

YMF754 (DS-1E) - same as YMF744B but less power consumption + S/PDIF In recording without resampling 44.1->48 kHz
[Datasheet for YMF754](https://www.dosdays.co.uk/media/yamaha/YMF754.pdf)

Gravis Ultrasound PnP
[Datasheet for Interwave chip](https://www.dosdays.co.uk/media/amd/InterWave%20Am78C201.pdf)

**Comment by sebastien64** [comment]
hello

please adds this graphics card

https://www.vgamuseum.info/index.php/cards/item/486-western-digital-wd90c31a-lr


thank

**Reply by TC1995** [reply]
in the plans, no ETA and it's got acceleration

**Comment by sebastien64** [comment]
the datasheet for this card

https://www.dosdays.co.uk/media/wd/[WD90C31_Datasheet.pdf](https://www.dosdays.co.uk/media/wd/WD90C31_Datasheet.pdf)

**Reply by TC1995** [reply]
yes, and no ETA when it will be implemented.

**Comment by SamuelV04** [comment]
To be fair I am not sure where to find documentation on things, but I'd like to see the Ich AC97 controller from Virtual Box and Qemu implemented. Given that it is emulated on those platforms I would hope that it wouldn't be too hard to add. And plus it isn't exactly tied to chipsets that are out of the emulator's scope so you could probably make it work. If the Analog Devices AD1881 device that you can emulate on some bords with an AMR slot serves this purpose then disregard.

**Reply by OBattler** [reply]
It is, the ICH AC'97 controller is part of the ICH chipset.

**Comment by sebastien64** [comment]
hello 

please add this card

https://theretroweb.com/expansioncards/s/asus-pci-sc875

thank

**Reply by TC1995** [reply]
That's just a plain 53c875.

**Reply by sebastien64** [reply]
this a scsi card from asus

**Reply by TC1995** [reply]
> this a scsi card from asus

And a plain 53c875 one we already emulate, the bios that is.

**Comment by sebastien64** [comment]
You do not accept this card.

**Comment by sebastien64** [comment]


**Reply by TC1995** [reply]
undocumented, no.

**Reply by jriwanek** [reply]
Also this discussion is for sound related devices, not video cards.

**Reply by TC1995** [reply]
As said, no.

**Reply by jriwanek** [reply]
As TC told you, it's undocumented, so the answer remains as he gave: NO.
Keep future posts on topic to the initial discussion, not begging for things you've already been told no, and given a reason why not, *especially* things which are off topic to the intial issue/discussion, in this case **sound cards**.

Besides, you've requested this particular card multiple times on this thread, so you're basically spamming at this point.

**Comment by sebastien64** [comment]
<img width="1280" height="1115" alt="1076" src="https://github.com/user-attachments/assets/444f8bd1-5d67-4e46-b08c-180d0335911b" />

**Comment by sebastien64** [comment]
Cancel this demand

**Reply by jriwanek** [reply]
We can't "cancel" it because we wern't gonna do it due to the reasons specified.
Now, if you could kindly stop spamming.

**Comment by AphisitC** [comment]
Add a New Soundcard - FortéMedia FM801 (-AS and -AU) Series

Audio Features for FM801-AS
• PCI v.2.2 compliance with bus master and scatter-
and-gathering capability
• Real DOS SoundBlaster Pro support
• Serial IRQ and PC/PCI support
• Built-in music synthesis for AdLib compatibility
• Built-in SRC (sampling rate converter) and digital
mixer
• AC-97 ver. 2.1 Compliance
• Multiple speakers, 2/4/6 channels support
• Dual game ports and Analog/Digital Joystick
Interface (Direct Input support)
• I2S ports and S/PDIF AC-3 raw data output
• Hardware push button volume control
• ACPI, PME, and PCI CLKRUN power
management
• MPU-401 port
• EEPROM interface support (24C00/01, 93C57)
• Professional 64 Voices Software WaveTable
supporting .MID (General MIDI) and .RMF (Rich
Music Format) files.
• QSound HSP dynamic 3-D positional audio (Just like the Trident 4DWave-NX, but with Lower CPU Utilization)
• Hardware push button volume control
(up/down/mute)
• 18 bits Audio CODEC support

Additional Features for FM801-AU
• S/PDIF AC-3 raw data input

[Product Brief for FM801-AS](https://theretroweb.com/chip/documentation/fm801-as-fortemedia-681697193e092999743257.pdf)
[Datasheet for FM801-AU](https://theretroweb.com/chip/documentation/fm801-au-fortemedia-65983e6e48742154702335.pdf)
[Hardware Design Specification for FM801 Series - (Rev. 2.2)](https://theretroweb.com/chip/documentation/fm801-linux-681697c5de04b016025005.pdf)
